packages/grafana-ui/src/components/TimePicker/TimeRangePicker/TimePickerContent.test.tsx

```diff
@@ -1,59 +1,174 @@
+import { dateTime, TimeRange } from '@grafana/data';
+import { render, RenderResult } from '@testing-library/react';
 import React from 'react';
-import { shallow } from 'enzyme';
 import { TimePickerContentWithScreenSize } from './TimePickerContent';
-import { dateTime, TimeRange } from '@grafana/data';
 
 describe('TimePickerContent', () => {
-  it('renders correctly in full screen', () => {
-    const value = createTimeRange('2019-12-17T07:48:27.433Z', '2019-12-18T07:48:27.433Z');
-    const wrapper = shallow(
-      <TimePickerContentWithScreenSize
-        onChangeTimeZone={() => {}}
-        onChange={value => {}}
-        timeZone="utc"
-        value={value}
-        isFullscreen={true}
-      />
-    );
-    expect(wrapper).toMatchSnapshot();
-  });
+  const absoluteValue = createAbsoluteTimeRange('2019-12-17T07:48:27.433Z', '2019-12-18T07:49:27.433Z');
+  const relativeValue = createRelativeTimeRange();
+  const history = [
+    createAbsoluteTimeRange('2019-12-17T07:48:27.433Z', '2019-12-17T07:49:27.433Z'),
+    createAbsoluteTimeRange('2019-10-18T07:50:27.433Z', '2019-10-18T07:51:27.433Z'),
+  ];
+
+  describe('Wide Screen', () => {
+    it('renders with history', () => {
+      const container = renderComponent({ value: absoluteValue, history });
```

Comment:
Is there a specific reason to use `container` here? The preferred way is to use `screen` for queries instead. 
```
renderComponent({ value: absoluteValue, history });
expect(screen..queryByText(/recently used absolute ranges/i)).toBeInTheDocument();
```

Reply:
I used screen initially but I thought container would make more sense for some reason. Fixed.